Gruntfile.js

```diff
@@ -117,7 +117,7 @@ module.exports = function(grunt) {
                 beautify: {
                     semicolons: false //Required: prevents dojo parser errors w/ minified files in this project
                 },
-                preserveComments: 'some',
```

Comment:
Hi Rob, please change this to: `preserveComments: /^!/` and then rerun `grunt build`. That should fix the issue.

Apparently the `'some'` option no longer works in the latest versions uglify. I tested the new pattern `v1.0.0` and it worked like a champed.